src/css/book.css

```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;700&display=swap');
+
+.bookCard {
+  width: 80%;
+  height: 10.625rem;
+  margin: 2.313rem auto;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  display: flex;
```

Comment:
- [ ] You are repeating the `display: flex` property 6 times, kindly fix this issue, remember that is better practice to repeat classes instead of properties. 😄 
    - For example, you could have a `d-flex` class that will have the `display: flex` then use this class in the elements that need it. 🤓